### DISCUSSION TITLE
Restore/Load data from Wal after Milvus restart.

### DISCUSSION THREAD

**Discussion by Yagyansh** [milvus-io/milvus#4685] [Q&A and General discussion]
Hi. I am running standalone Milvus version 0.10.0 on a virtual machine. I have enabled the wal & I had inserted around 1200000 vectors to Milvus. I restarted my Milvus service because of some configuration changes, after the restart, all the vector data is gone, I no longer have any vector in Milvus. Is there any way to load or restore the data from the wals or via some other method?

**Comment by yhmo** [comment]
All milvus data is under a "/milvus/db" folder, including the wal and vectors data. The folder is specified by the server_config.yaml file:

```
storage:
  path: /var/lib/milvus
```

I think the change of configuration re-mapping the path to another folder, so you think the vector data is gone. 
And docker command also can mapping this folder from container to outside directory, like this:
`docker run -it -d -p 19530:19530 -v /xxx/xxx/milvus/db:/var/lib/milvus/db -v /xxx/xxx/milvus/conf:/var/lib/milvus/conf -v /xxx/xxx/milvus/logs:/var/lib/milvus/logs -v /xxx/xxx/milvus/wal:/var/lib/milvus/wal milvusdb/milvus:0.10.2-cpu-d081520-8a2393`

So, you need to remember what directory the path mapping to, otherwise, milvus could not know where to read your vector data.
